subgraph/package.json

```diff
@@ -6,14 +6,17 @@
   "scripts": {
     "generate-subgraph-yaml": "node ./bin/subgraph_generator.js",
     "codegen": "graph codegen",
-    "graph": "graph",
-    "build": "node ./bin/build",
-    "deploy": "node ./bin/deploy",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ unlock-protocol/goerli-v2",
```

Comment:
We should not make these commands network specific... (if anything we should have a command to deploy _all_ networks).

Reply:
yep thats a dev artefact... will remove